dts/bindings/sensor/st,stm32-temp-cal-c0.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2022, Benjamin Björnsson <benjamin.bjornsson@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    STM32 family TEMP node for production calibrated sensors like C0
+    with a single calibration temperature.
+
+compatible: "st,stm32-temp-cal-c0"
```

Comment:
To follow naming scheme usually adopted for stm32, use `st,stm32c0-temp-cal`

Reply:
As was agreed above.. thanks for spotting this. 👍